### DISCUSSION TITLE
Reset connections

### DISCUSSION THREAD

**Discussion by szytko** [gopro/labs#334] [General]
I have GoPro 10 connected over USB to Raspberry PI and sometimes it is losing connection for unknown reason, camera is not listed by `lsusb` command, rebooting Raspberry PI also doesn't fix the connection. Only manually going in the camera to Reset connections menu is helping in this case. Is there any other way to reset connections ? Like through WiFi/BLE or Quick app ? Even  doing that using QrCode would be really helpful in case of the setup I'm working on.

**Comment by dnewman-gpsw** [comment]
There like is via WiFi/BLE, a good question for the Open GoPro folks at https://gopro.github.io/OpenGoPro/

In Labs the equivalent to the connections reset is the QR Code !WRESET, it requires three button presses to confirm.  oMNAKE=1!WRESET will do that same with presses.  But that might not be what you need, as it does reboot the camera.

**Reply by szytko** [reply]
I tried to scan `oMNAKE=1!WRESET` and camera rebooted afterward (reboot is fine for my setup). Will try to reproduce USB connection issue to see if this helps. Shall I drop question at OpenGoPro github how to do it through WiFi/BLE ?
Great thanks for help!
